docs/source/TestParameters.rst

```diff
@@ -0,0 +1,945 @@
+===============
+Test parameters
+===============
+
+.. note:: This section describes in detail what are test parameters and how
+   the whole variants mechanism works in avocado. If you're interested in
+   basics, see :ref:`accessing-test-parameters` or practical view by examples
+   in `Yaml_to_mux plugin`_.
+
+.. warning:: The multiplexer is under heavy refactor and some of the APIs
+   might still change in the following months (written on 2016-01-22),
+   then we'll do our best to keep the public interfaces as stable as
+   possible.
+
+Avocado allows passing parameters to each executed test or even running
+several variants of each test. These parameters are available in (test's)
+``self.params`` and are of :mod:`avocado.core.varianter.AvocadoParams` type.
```

Comment:
well is there a real difference, or is it just "better"? Anyway I'm fixing it, just seeing no real difference.